### DISCUSSION TITLE
Any ideas on next webdriverio release?

### DISCUSSION THREAD

**Discussion by shreyanshversion1** [webdriverio/webdriverio#12447] [Q&A]
Hi all,
@christian-bromann I know wdio is already very frequent with the releases and bug fixes.
I wanted to check if there is an upcoming release as there is a bugfix https://github.com/webdriverio/webdriverio/issues/12125
related to LambdaTest integration that I need to implement with my project.

Thanks.

**Comment by christian-bromann** [comment]
This fix has been released as part of [v8.32.2](https://github.com/webdriverio/webdriverio/commit/dd4ccd4bcbca4b5f97af81fc6ccee11cb8859654)
